internal/resume/resume_test.go

```diff
@@ -21,6 +21,7 @@ func TestResumeDeserialization(t *testing.T) {
 
 	expectedTime := time.Date(2004, 01, 1, 0, 0, 0, 0, time.UTC)
 	expected := resume.Resume{
+		CensoringEnabled: false,
```

Comment:
Gonna need to check the value as well, with `g.Expect(r.CensoringEnabled).To(Equal(expected.CensoringEnabled))`

Reply:
Woops